index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/cf2aa695e2.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Nanum+Gothic+Coding&display=swap" rel="stylesheet">
+  <title>GT Hackathon</title>
+</head>
+
+<body>
+  <header>
+    <div class="wrap topmenu_wrap">
+      <nav class="top_menu">
+        <ul>
+          <li><a href="#"><i class="fa-brands fa-facebook"></i></a></li>
+          <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+          <li><a href="#" target="_blank">English</a></li>
+          <li><a href="#" target="_blank">logout</a></li>
+        </ul>
+      </nav>
+    </div>
+    <nav class="navbar">
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <a href="#" class="logo">GTH.</a>
+
+      <ul class="nav-menu">
+        <li class="nav-item"><a href="#about" active class="nav-link">Event Introduction</a></li>
+        <li class="nav-item"><a href="#program" class="nav-link">Main Program</a></li>
+        <li class="nav-item"><a href="#speakers" class="nav-link">Featured</a></li>
+        <li class="nav-item"><a href="#partners" class="nav-link">Partners</a></li>
+        <li class="nav-item"><a href="#index.html" class="live nav-link">Register</a></li>
+      </ul>
+
+
+    </nav>
+  </header>
+
+  <section class="about" id="about">
+    <div class="container">
+      <div class="topText">
+        <h3>"Hello! Tech world"</h3>
+      </div>
+      <div class="title">
+        <h1>GLOBAL-TECH</h1>
+        <h1>HACKATHON</h1>
+        <h1>2022</h1>
```

Comment:
- [ ] For the mobile version, kindly adjust `margin-top` of the headline. It looks way too big compared to the design guidelines.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/198384156-31bb3281-fafc-4e12-a3fe-e2ca54fdcfed.png)

## Design guidelines:
![image](https://user-images.githubusercontent.com/50721479/198384194-ea63aeac-ba9b-42df-aff3-b484f32583d7.png)